src/components/LeaderboardBtn.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+const Btn = styled.div`
```

Comment:
Is this going to be a button? I thought it was a table row, is there any functionality planned? Maybe a link within the table would be enough if you are planning to add some redirecting?